src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
It's better to apply the 'box-sizing: border-box;' rule to all elements using the universal selector (*) instead of just applying it to the body element.